.github/workflows/ci.yml

```diff
@@ -249,3 +249,21 @@ jobs:
       - name: Install
         run: |
           cmake --build build --target install -j 2
+
+
+  scripts:
```

Comment:
If you pull this out into its own workflow, you could use a file restriction to only run it when there are edits to Scripts/Python files: https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#example-including-paths

Reply:
Good idea. I've split it out into a new workflow that only runs when either the python files or the workflow files change.